engine-rest/engine-rest-openapi/src/main/templates/paths/history/decision-instance/delete/post.ftl

```diff
@@ -0,0 +1,71 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/decision-instance/post-delete/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "deleteAsync"
+      tag = "Historic Decision Instance"
+      summary = "Delete Async (POST)"
+      desc = "Delete multiple historic decision instances asynchronously (batch).
+              At least `historicDecisionInstanceIds` or `historicDecisionInstanceQuery` 
+              has to be provided. If both are provided then all instances matching query 
+              criterion and instances from the list will be deleted."
+  />
+
+  <@lib.requestBody
+      mediaType = "application/json"
+      dto = "DeleteHistoricDecisionInstancesDto"
+      examples = ['"example-1": {
+                     "summary": "POST `/history/decision-instance/delete`",
+                     "description": "POST `/history/decision-instance/delete`",
+                     "value": {
+                       "historicDecisionInstanceIds": [
+                         "aDecision",
+                         "secondDecision"
+                       ],
+                       "historicDecisionInstanceQuery": {
+                         "decisionDefinitionKey": "a-definition-key"
+                       },
+                       "deleteReason": "a delete reason"
+                     }
+                   }']
+  />
+
+  "responses": {
+
+    <@lib.response
+        code = "200"
+        dto = "BatchDto"
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "POST `/history/decision-instance/delete`",
+                       "description": "POST `/history/decision-instance/delete`",
+                       "value": {
+                         "id": "aBatchId",
+                         "type": "aBatchType",
+                         "totalJobs": 10,
+                         "batchJobsPerSeed": 100,
+                         "jobsCreated": 10,
+                         "invocationsPerBatchJob": 1,
+                         "seedJobDefinitionId": "aSeedJobDefinitionId",
+                         "monitorJobDefinitionId": "aMonitorJobDefinitionId",
+                         "batchJobDefinitionId": "aBatchJobDefinitionId",
+                         "suspened": false,
+                         "tenantId": null
```

Comment:
The example is missing the `createUserId` property. If we decide to add it here, we should also add it in the docs. Feel free to decide for yourself.

Reply:
```suggestion
                         "tenantId": null,
                         "createUserId": "aUser"
```